docs/reference/query-dsl/rank-feature-query.asciidoc

```diff
@@ -120,31 +159,83 @@ GET test/_search
     }
   }
 }
---------------------------------------------------
+----
 // CONSOLE
 
-[float]
-=== Supported functions
 
-The `rank_feature` query supports 3 functions in order to boost scores using the
-values of rank features. If you do not know where to start, we recommend that you
-start with the `saturation` function, which is the default when no function is
-provided.
+[[rank-feature-top-level-params]]
+==== Top-level parameters for `rank_feature`
+
+`field`::
+(Required, string) <<rank-feature,`rank_feature`>> or
+<<rank-features,`rank_features`>> field used to boost
+<<query-filter-context,relevance scores>>.
+
+`boost`::
++
+--
+(Optional, float) Floating point number used to decrease or increase
+<<query-filter-context,relevance scores>>. Defaults to `1.0`.
+
+Boost values are relative to the default value of `1.0`. A boost value between
+`0` and `1.0` decreases the relevance score. A value greater than `1.0`
+increases the relevance score.
+--
+
+`saturation`::
++
+--
+(Optional, <<rank-feature-query-saturation,function object>>) Saturation
+function used to boost <<query-filter-context,relevance scores>> based on the
+value of the rank feature `field`. If no function is provided, the `rank_feature`
+query defaults to the `saturation` function. See
+<<rank-feature-query-saturation,Saturation>> for more information.
+
+You cannot use the `saturation` parameter with the `log` or `sigmoid`
```

Comment:
Small comment, perhaps it'd be clearer to say "Only one function `saturation`, `log`, or `sigmoid` can be provided".

Reply:
Thanks for the suggestion. I changed this sentence with 22352b0.